<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ЦОСиИ лаб. 4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="sample0ImageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="testImageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sampleLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Samples:</string>
    </property>
   </widget>
   <widget class="QLabel" name="testLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Test image:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recognizeButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recognize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseSampleButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open folder ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseTestButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file ...</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="noiseSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string>%</string>
    </property>
    <property name="prefix">
     <string>Noise </string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="adjustButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Adjust</string>
    </property>
   </widget>
   <widget class="QLabel" name="sample1ImageLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sample2ImageLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="similarity0Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="similarity1Label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="similarity2Label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sample3ImageLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sample4ImageLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="similarity3Label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="similarity4Label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="resultImageLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>It looks like:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
